client/src/components/History/HistoryBadge/HistoryBadge.module.scss

```diff
@@ -11,5 +11,5 @@
 }
 
 .disabled {
-    background-color: var(--global-color-accent--light);
+    background-color: var(--global-color-accent--weak);
 }
```

Comment:
__What is a disabled History Badge?__

It is a way to make history badge color match the "greyed out" appearance of a disabled Sidebar. Such a state would not be necessary for History Badge if the disabled Sidebar styles was `opacity: 0.N` instead of [`color: …`](https://github.com/TACC/Core-Portal/commit/300421e#diff-18d0e6596b231f62ab096d82b9b1836d7c86fd5a0a51e6ab34c9b9ba3b261c98R13-R15).